initfiles/etc/DIR_NAME/environment.xml.in

```diff
@@ -898,6 +898,17 @@
              name="s1"
              netAddress="."/>
   </EclSchedulerProcess>
+  <TopoServerProcess build="_"
+                     buildSet="toposerver"
+                     daliServers=""
+                     description="Topology Server process"
+                     name="toposerver"
+                     port="9004">
+   <Instance computer="localhost"
+             directory="/var/lib/HPCCSystems/toposerver"
```

Comment:
Should probably be  directory="${RUNTIME_PATH}/toposerver"
Incidentally, the spark thor entry should be the same.

Reply:
Fixed both to use RUNTIME_PATH